src/app/home-page/home-page.controller.js

```diff
@@ -10,14 +11,17 @@ export default class HomePageController {
 
     search() {
         this.latestSearchesUtil.addCity(this.city);
-        //Will do this call on the search page
-        //this.openWeatherMaps.getFiveDayForecastCity(this.city);
+        this.openWeatherMaps.getCurrentForecastCity(this.city);
+        this.openWeatherMaps.getFiveDayForecastCity(this.city);
```

Comment:
This logic should be called on the init method of the search component, because if the user refresh the page (or access directly) on /search, the current and future forecast won't be available.
If you have doubts don't hesitate to contact me :)

Reply:
Sure, I undertand, messed up everything a bit, changing it right now